package.json

```diff
@@ -57,6 +57,8 @@
     "start": "node app.js",
     "test": "karma start --single-run",
     "jshint": "jshint platform example",
+    "lint": "./node_modules/gulp/bin/gulp.js lint",
+    "checkstyle": "./node_modules/gulp/bin/gulp.js checkstyle",
```

Comment:
What's the reason for adding these?

Reply:
It's for Developer experience. Cannot use gulp unless it's installed globally, which is not the case for many. Could remove it if it's not preferable?

Reply:
I do agree with using `npm run` over `gulp`; it is better if we do not rely on global installs. If people want to use `gulp-cli`, then so be it.

Reply:
I'm quite sure that run scripts use local commands, so there's no need for the `./modules/...` bit.

```
$ npm run which

> prottapp@0.0.0 which /Users/.../projects/prott/prott-webapp
> which grunt

/Users/.../projects/prott/prott-webapp/node_modules/.bin/grunt
```

<img width="179" alt="2018-05-11 15 16 40" src="https://user-images.githubusercontent.com/11208360/39909583-5d469e6e-552e-11e8-8db6-4f478d552c3d.png">

Reply:
Yup, can do that. Should we have one script for each command, or a chain? I think it's possible to run `gulp` and it will chain everything.